src/Lecture2.hs

```diff
@@ -41,7 +41,7 @@ module Lecture2
     ) where
 
 -- VVV If you need to import libraries, do it after this line ... VVV
-
+import Text.Read
```

Comment:
I see a GHC warning that this important is redundant. Maybe you used it before but haven't cleaned up? 🤔